src/Fantomas/CodePrinter.fs

```diff
@@ -1899,7 +1899,7 @@ and genExpr astContext synExpr ctx =
 
                 isShortExpression ctx.Config.MaxDotGetExpressionWidth (genExpr sepNone) (genExpr sepNln) ctx
 
-        | AppTuple (e, lpr, args, rpr) when (not astContext.IsInsideDotGet) ->
+        | AppTuple (e, lpr, args, rangeOfTuple, rpr) when (not astContext.IsInsideDotGet) ->
```

Comment:
I have used `rangeOfTuple` as suggested but I have seen you are using both `rangeOfXXX` and `XXXRange` notation in the code so let me know if you prefer the other.

Reply:
Hmm, good remark. Perhaps go for `tupleRange` instead.
Both do appear in the code base indeed, this hasn't really been a conscious decision I think.